packages/material-ui/src/Select/Select.js

```diff
@@ -81,12 +97,12 @@ const Select = React.forwardRef(function Select(inProps, ref) {
             SelectDisplayProps: { id, ...SelectDisplayProps },
           }),
       ...inputProps,
-      classes: inputProps ? deepmerge(classes, inputProps.classes) : classes,
+      classes: inputProps ? deepmerge(otherClasses, inputProps.classes) : otherClasses,
       ...(input ? input.props.inputProps : {}),
     },
     ...(multiple && native && variant === 'outlined' ? { notched: true } : {}),
     ref,
-    className: clsx(className, InputComponent.props.className),
+    className: clsx(classes.root, InputComponent.props.className, className),
```

Comment:
`className` should be the last.

Reply:
It doesn't matter

Reply:
A side note, in the dev tools, Chrome seems to keep the same order in the DOM view:

<img width="441" alt="Screenshot 2021-05-12 at 11 05 13" src="https://user-images.githubusercontent.com/3165635/117948977-f28bdb80-b311-11eb-82ab-4b807e451e20.png">

and to sort ASC in the .cls view

<img width="593" alt="Screenshot 2021-05-12 at 11 05 42" src="https://user-images.githubusercontent.com/3165635/117949039-02a3bb00-b312-11eb-8cf9-4fcd1a8f006b.png">

Reply:
As for "className should be last", no idea if it has any impact on the developers. Based on how Chrome handles it, maybe it would be slightly better if it's first? Not sure I guess the only value of being last is to do as in the other components, not to have to think about where it should be 🤷‍♂️

Reply:
Agree, my comment was that it doesn't matter for the behavior. Let's do it as it is in the other components.

Reply:
Yeah, the order in the `className` doesn't matter. What matters is the order that the styles are defined in the stylesheet. I put it as the last just to keep the same convention that exist in other components.